.github/workflows/create-draft-release.yml

```diff
@@ -1,26 +1,36 @@
-name: Create Release
+name: Create or Update Draft Release
 
 on:
   push:
     branches:
     - main
+    - v*
+  repository_dispatch:
+    types: [ version-bump ]
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version of the release to cut (e.g. 1.2.3)'
+        required: false
+
+concurrency: release
 
 jobs:
   unit:
-    name: Run Tests
+    name: Unit Tests
     runs-on: ubuntu-latest
     steps:
     - name: Setup Go
       uses: actions/setup-go@v3
       with:
         go-version: 1.18.x
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
     - name: Setup Docker Multi-Platform Builds
       run: |
         docker run --privileged --rm docker/binfmt:a7996909642ee92942dcd6cff44b9b95f08dad64
-    - name: Run Tests
-      run: go test -v -count=1 ./...
+    - name: Run Unit Tests
+      run: ./scripts/unit.sh
```

Comment:
there is no unit test script, this should stay as `go test` in all instances in this PR

Reply:
It's my intent to introduce `./scripts/unit.sh` with this PR, that was accidentally excluded but will be included with another commit.

I do think it's helpful to standardize on `./scripts/<do-thing>.sh` for consistency. I'm not in this repo very much so my finger memory fails me when the script is missing.